### DISCUSSION TITLE
Distributed mMIMO and Cell-Free mMIMO in Sionna

### DISCUSSION THREAD

**Discussion by pablosreyero** [NVlabs/sionna#243] [Q&A]
Hi,

I have been playing with the simulator for quite a while, and I wanted to know if Distributed mMIMO and Cell-Free is on your development plans of Sionna, since what I've seen from all discussions and tutorials, it is not natively implemented yet. The reason of my question lies on a potential manual implementatipon by myself, but first, I wanted to be sure if this approach was realistic or if you guys were planning to release a Sionna update including this features. Do you guys think this kind of manual approach could be succesfull realistically speaking?

Thanks in advance!
Pablo.-

**Comment by jhoydis** [comment]
Hi Pablo,

What feature are you missing to implement cell-free MIMO in Sionna? Is it just being able to to generate channel impulse responses for a distributed MIMO array, or something else?

**Reply by pablosreyero** [reply]
Hi Jakob,

Sorry for my late reply, this last months I've been focusing more on other projects, but I have looked into the source code, and checked for the implementation I asked you about, and indeed there's no need for additional tools/features, I think it was just a matter of dig into the code and understand your implementation. Nevertheless, thank you very much for your quick reply. 

Cheers,
Pablo.-
